lib/ui/views/installer/installer_viewmodel.dart

```diff
@@ -199,8 +199,8 @@ class InstallerViewModel extends BaseViewModel {
 
       '\n~ Settings',
```

Comment:
Also, the weird `~` symbol should be replaced with something more traditional such as `-`.